lib/service/base.rb

```diff
@@ -85,6 +85,10 @@ def method_missing(method_name, *args, &block)
         return super if args.present?
         store[method_name]
       end
+
+      def respond_to_missing?(name, include_all)
```

Comment:
This was missing and prevented RSpec matcher `have_attributes` from working properly.